### PR TITLE
Publish stable version to bintray

### LINKED ISSUE #25: Publish stable version of hidden-args somewhere accessible via sbt
We came across the following problem during our release process: https://github.com/morganey-lang/Morganey/issues/271

We suspect that publishing a stable version of hidden-args somewhere could help us to resolve it easily. Is it possible to do that? Thanks!

### PR DESCRIPTION
**Pull request by keddelzz**
closes #25

### CODE REVIEW

File: project/build.properties [hunk 1]
```diff
@@ -1 +0,0 @@
-sbt.version = 0.13.12
```
Comment (ForNeVeR) [inline project/build.properties:1]: Why are you removing that?

**Review by ForNeVeR — APPROVED** [review]
Overall: LGTM.

File: project/build.properties (hunk 1 above)
Comment (keddelzz) [reply project/build.properties:1]: I don't think it is necessary to explicitly specify a sbt-version in this project.

Comment (ForNeVeR) [reply project/build.properties:1]: Do you have any particular problem with the fixed sbt version, or is it just a general intention? I'm okay with the change; just curious.

Comment (keddelzz) [reply project/build.properties:1]: No, i don't have a particular problem with the fixed version, but as far as I understand it's used to force a minimum required sbt-version for your build. As `hidden-args` doesn't really requires this, I removed the fixed version.

Comment (ForNeVeR) [reply project/build.properties:1]: Alright!

**Comment by keddelzz** [comment]
Thank you very much for reviewing the PR, @ForNeVeR!

Merging it now!

**Pull request merged**